manifest.json

```diff
@@ -15,10 +15,10 @@
   },
   "externally_connectable": {
     "matches": [
-      "http://*/*",
-      "https://*/*"
+      "*://*.b12.io/*"
```

Comment:
haha my bad---maybe just add a "how to customize" section to the README and point to this config and load/save apis as a bullet points without too much detail